site/src/docs/patterns/login/building-login.mdx

```diff
@@ -0,0 +1,110 @@
+---
+slug: '/patterns/login/building-login'
+title: 'Building login'
+navTitle: 'Building login'
+---
+
+import LeadParagraph from '../../../components/LeadParagraph';
+import Image from '../../../components/Image';
+import StatusLabel from '../../../components/StatusLabel';
+import {
+  Breadcrumb,
+  Button,
+  Header,
+  Hero,
+  Navigation,
+  SideNavigation,
+  Tabs,
+  Footer,
+  IconCheck,
+  IconError,
+  IconFacebook,
+  IconTwitter,
+  IconInstagram,
+  IconYoutube,
+  IconTiktok,
+  IconLinkedin,
+  Logo,
+  logoFi,
+} from 'hds-react';
+
+# Building login
+
+<StatusLabel type="error">Draft</StatusLabel>
+
+<LeadParagraph>
+  The Login pattern describes how HDS Login components and HDS Header login functionalities are meant to work together.
+</LeadParagraph>
+
+## Principles
+
+### Login components
+
+<InternalLink href="/components/login">HDS Login</InternalLink> components include React context, components and hooks for
+handling user authorisation, API tokens, session polling and GraphQL queries.
+
+#### Requirements
+
+The following steps are required for user authentication:
+
+- The url of the service connecting to the OIDC server <InternalLink href="/components/login/usage/#loginprovider">must be registered with a scope and a client ID</InternalLink>.
+- The callback from the OIDC server should be handled with the <InternalLink href="/components/login/usage/#logincallbackhandler">LoginCallbackHandler</InternalLink> component.
+
+#### Log in process
+
+Logging in or out of an OIDC server requires browser redirections to the server and back. UI's only show login and logout buttons. No other form elements are needed.
```

Comment:
Better readability if we write it open. The user interface only shos login and logout buttons.